users/ehudec/index.md

```diff
@@ -5,14 +5,14 @@ author: Eric Hudec
 ---
 # Eric Hudec
 
-<img src="images/Ashlea and I.JPEG" height="403" width="302">
---------------
+## email: [hudece@oregonstate.edu](mailto:hudece@oregonstate.edu) | cell: 217-341-5975
 
-Contact: [hudece@oregonstate.edu](mailto:hudece@oregonstate.edu)
+<img src="images/selfie_y2.jpeg" height="300" width="300"> <img src="images/porter.jpeg" height="300" width="300">
+--------------
 
 
-Hello! I am a first year graduate student working in the Radiation Transport and Reactor Physics research group under the direction of Dr. Todd Palmer at Oregon State University. 
+I am a second year graduate student working in the Radiation Transport and Reactor Physics research group under the direction of Dr. Todd Palmer at Oregon State University. As a part of my research assistantship, I work with the Nuclear Criticality Safety Division at Lawrence Livermore National Lab, as an intern in the Integral Experiments group. My work with LLNL has primarily focused on the analysis of a benchmark experiement involving high-multiplication subcritical assemblies, including modelling the experimental setup in MCNP and performing uncertainty analysis. From July to December of 2023 I am working full time on this project to prepare a benchmark document for publication in 2024. The results from this experiment will also serve as the basis for my Master's Thesis, which will (hopefully) be complete in the summer of 2024.
```

Comment:
"experiement" should be experiment, also I thought "modelling" was wrong but apparently that's ok